bigbluebutton-html5/private/locales/en.json

```diff
@@ -25,8 +25,8 @@
   "app.userlist.menu.kickUser.label": "Kick user",
   "app.userlist.menu.muteUserAudio.label": "Mute user",
   "app.userlist.menu.unmuteUserAudio.label": "Unmute user",
-  "app.userlist.useritem.nostatus.arialabel": "{username} {presenter} {you}",
-  "app.userlist.useritem.status.arialabel": "{username} {presenter} {you} current status {status}",
+  "app.userlist.useritem.nostatus.arialabel": "{0} {1} {2}",
```

Comment:
If theres is nothing to translate we dont need those strings here. You can just use string literals to concatenate the values.

Already talked with @KDSBrowne on Slack.